src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Commands.RecoveryServices.Backup.Helpers.csproj

```diff
@@ -40,9 +40,8 @@
     <Reference Include="Microsoft.Azure.Common">
```

Comment:
We will keep all of these in azure sql branch. and will release from Azure SQL branch. This effort is to make sure there is no surprise later for merging.